versions-manifest.json

```diff
@@ -1,4 +1,38 @@
 [
+  {
+    "version": "3.10.0-beta.1",
```

Comment:
Why is this called beta 1?  Building CPython from their main branch still builds `3.10.0a7+`.  This is misleading and confusing.

Reply:
This version was built based on source code from https://www.python.org/ftp/python/3.10.0/Python-3.10.0b1.tgz that looks like beta 1

Reply:
I don't know what that tgz, but: there's no b1 on the python.org downloads page, and the tip of master still builds a7+.  The official PEP says b1 will happen soon, but not yet: https://www.python.org/dev/peps/pep-0619/

Can you address this on this issue? https://github.com/actions/setup-python/issues/207

Whatever process created this beta1 label, it was incorrect.

Reply:
@maxim-lobanov what alerted you to the presence of that tarball? (BTW: it's been removed now).  Seems like this project needs a different trigger to create releases here.

Reply:
@MaksimZhukov will take care about removing release

Reply:
Some important fact to consider: some rare cases the initial tarballs for Linux are released in the FTP server but some problem is detected with macOS or Windows builds and the release will be halted/restarted. In general what points to the official release being valid is the announcements and the presence of the page in the download section of python.org.

The web has an API to query the objects that is going to probably be used by the folks that create containers automatically, so maybe that's a good candidate.

Please, contact me or any member of the release team if you are interested in helping on this area.